package.json

```diff
@@ -13,7 +13,7 @@
     "types"
   ],
   "repository": "jshttp/mime-types",
-  "dependencies": {
+  "peerDependencies": {
     "mime-db": "1.52.0"
```

Comment:
is the intention here really to pin? if so, i'd use an explicit `=`, but using `^` is the better approach here so that security updates and bugfixes can be used without use of `overrides`.

Reply:
Ugh, good catch.  I'd actually changed this to "1.x" ... 'just forgot to push it. 😛

Reply:
I would strongly suggest avoiding that syntax, and instead using `^1.52.0` or `^1.0.0` if that's really what you want.

Reply:
Interesting.  How come?   Is it because "1.x" doesn't let you specify a minimum version other than the implied "1.0.0"?